src/cdk/dialog/dialog-container.ts

```diff
@@ -112,7 +116,21 @@ export class CdkDialogContainer<C extends DialogConfig = DialogConfig>
     this._document = _document;
 
     if (this._config.ariaLabelledBy) {
-      this._ariaLabelledByQueue.push(this._config.ariaLabelledBy);
+      this._addAriaLabelledBy(this._config.ariaLabelledBy);
```

Comment:
nit: since this is in the constructor, maybe we can push to the array directly without having to call `markForCheck`?

Reply:
SGTM